src/osut/osut.py

```diff
@@ -7411,7 +7454,7 @@ def addSkyLights(spaces=[], opts=dict) -> float:
             if srr > 0.90:
                 oslg.log(CN.WRN, "SRR (%.2f) reset to 90% (%s)" % (srr, mth))
 
-            srr = numpy.clip(srr, 0.00, 0.10)
+            srr = clamp(srr, 0.00, 0.10)
```

Comment:
Similar function call/arguments as _Numpy_'s `clip`. Haven't added unit stress tests, yet all higher-level functions work as intended. Good enough for now.